tst/testinstall/kernel/blister.tst

```diff
@@ -209,5 +209,59 @@ true
 gap> MEET_BLIST(x, [true,false,false,false]);
 false
 
+# FuncFLIP_BLIST
+gap> FLIP_BLIST(fail);
+Error, FlipBlist: <blist> must be a boolean list (not the value 'fail')
+gap> x:= [false,true,true,false];;
+gap> FLIP_BLIST(x);
+gap> x;
+[ true, false, false, true ]
+gap> FLIP_BLIST(x);
+gap> x;
+[ false, true, true, false ]
+gap> for i in [0..200] do
+> f1 := List([1..i], x -> false);
+> f2 := List([1..i], x -> false);
+> t1 := List([1..i], x -> true);
+> t2 := List([1..i], x -> true);
```

Comment:
Note a request for changes, just a general remark: in real code, I'd always suggest to write `ListWithIdenticalEntries(i, true);` instead, which is optimized and directly produces a blist (unlike the above code, which produces a plist).

Reply:
Yup, me too, I think it was @ChrisJefferson that wrote this particular line.

Reply:
I keep forgetting about ListWithIdenticalEntries, there is no good reason not to use it.